crates/evm/src/tests/sys_tx_tests.rs

```diff
@@ -804,7 +805,7 @@ fn test_change_upgrade_owner() {
     // Assert if hash is equal to 0xdeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddead
     assert_eq!(
         hash,
-        reth_primitives::Bytes::from_str(
+        alloy_primitives::Bytes::from_str(
```

Comment:
Well, you can replace it with `b256!("deaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddeaddead");`